frontend/fingertips-frontend/playwright/page-objects/pages/resultsPage.ts

```diff
@@ -4,6 +4,7 @@ import { expect } from '../pageFactory';
 export default class ResultsPage extends BasePage {
   readonly resultsText = 'You searched for indicator';
   readonly backLink = 'search-results-back-link';
+  readonly searchResult = 'search-result';
 
   async checkSearchResults(searchTerm: string) {
```

Comment:
Can the `checkSearchResults` method now be removed, given you've added `checkResultsAreDisplayed`?